### PR TITLE
Added create account button to user dash, improved mobile styling

### PR DESCRIPTION
**Pull request by tkim516**
### Pull Request Summary
- Added create account button so that it is even easier for a user to create a new account
- Changed some text
- Improved styling for mobile

### Modifications
.UserDashboard.tsx

### Testing Considerations
Tested for various screen sizes

### Pull Request Checklist

- [ ] Code is neat, readable, and works
- [ ] Comments are appropriate
- [ ] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [ ] The developer name is specified
- [ ] The summary is completed
- [ ] Assign reviewers

### Screenshots/Screencast

Screenshot 2024-05-19 at 11 19 33 PM

Screenshot 2024-05-19 at 11 19 21 PM

### CODE REVIEW

**Comment by tkim516** [comment]
Just commited changes for navbar and admin navbar

File: src/app/components/UserDashboard.tsx [hunk 1]
```diff
@@ -367,35 +372,62 @@ const handleButtonClickToStopPropogation = (event: React.MouseEvent<HTMLButtonEl
              <Flex
                flexDirection={"column"}
                alignItems={"center"}
+               justifyContent={"center"}
              >
                <Text
-                 fontSize="2xl"
-                 fontWeight="bold"
+                 fontSize={["l","l","xl"]}
+                 fontWeight="600"
```
Comment (javalosr2004) [inline src/app/components/UserDashboard.tsx:379]: thicky

File: src/app/components/UserDashboard.tsx [hunk 2]
```diff
@@ -367,35 +372,62 @@ const handleButtonClickToStopPropogation = (event: React.MouseEvent<HTMLButtonEl
...
                  color="black"
                  textAlign="center"
-                 mt="60px"
+                 mt="8"
                >
-                 Sign in to see all your upcoming events！
+                 Create an account or sign in to see your upcoming events!
```
Comment (javalosr2004) [inline src/app/components/UserDashboard.tsx:384]: yes!

File: src/app/components/UserDashboard.tsx [hunk 3]
```diff
@@ -367,35 +372,62 @@ const handleButtonClickToStopPropogation = (event: React.MouseEvent<HTMLButtonEl
...
-                 <Button
-                   width="200px"
-                   colorScheme="yellow"   
-                   mt="5"
-                 >
-                   Sign in
-                 </Button>
-               </Link>
+               <Box
+                flexDirection={["column", "column", "row"]} // for breakpoints of 0px, 480px, and 768px
```
Comment (javalosr2004) [inline src/app/components/UserDashboard.tsx:387]: I sure hope nobody is using 0px

File: src/app/styles/navbar/navbar.module.css [hunk 4]
```diff
@@ -12,13 +12,17 @@
     align-items: center;
     height: 100%;
     position: relative;
+    font-family: "Lato";
```
Comment (javalosr2004) [inline src/app/styles/navbar/navbar.module.css:15]: less performant :( causes duplicate download of font.

File: src/app/styles/navbar/navbar.module.css [hunk 5]
```diff
@@ -12,13 +12,17 @@
...
+    font-weight: 600;
+    max-width: 100%;
 }
 
 .navbar {
     height: 60px;
     background-color: #2b2b33;
-    position: relative;
+    position: sticky;
```
Comment (javalosr2004) [inline src/app/styles/navbar/navbar.module.css:23]: not sure if its been fixed, but as of now mobile view if you click on empty space the navbar will not close, which kinda sucks, but tbf i don't know how many times they will be using mobile.

**Review by javalosr2004 — APPROVED** [review]
Good job tyler! Nice improvements and usage of Chakra built in breakpoints!

File: src/app/styles/navbar/navbar.module.css (hunk 4 above)
Comment (tkim516) [reply src/app/styles/navbar/navbar.module.css:15]: Will fix!

**Pull request merged**